src/components/card/playlist-category-footer/PlaylistCategoryFooter.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classes from './PlaylistCategoryFooter.module.css';
+import { useAsync } from '../../../hooks/useAsync';
+import DeleteIcon from '../../../assets/svg-icons/DeleteIcon';
+import { detelePlaylist } from '../../../utils/video-utils';
```

Comment:
I think there's a type in `deletePlaylist` word. You might want to correct that if not intentional.